pkg/operator/api/userconfig/config.go

```diff
@@ -37,15 +37,21 @@ var typeFieldValidation = &cr.StructFieldValidation{
 }
 
 func (config *Config) Validate(projectBytes []byte) error {
-	if err := config.App.Validate(); err != nil {
-		return err
-	}
+	err := config.App.Validate()
 
-	projectFileMap, err := zip.UnzipMemToMem(projectBytes)
 	if err != nil {
 		return err
 	}
 
+	projectFileMap := map[string][]byte{}
```

Comment:
nit: for no strong reason, just as convention we've been using the `make()` syntax